src/command/use_.rs

```diff
@@ -1,50 +0,0 @@
-// Rust doesn't allow using keywords as module names so we have to call this `use_`.
```

Comment:
Related to the above comment, is there a way in StructOpt to have "undocumented" commands that parse correctly but aren't included in the help message? Specifically, when we renamed from `use` to `pin`, we left this command around with the help message that guides the user to the correct subcommand based on what they are trying to do. Is there a way to recreate that behavior?

Reply:
There is, and I've used a `macro_rules`-style macro so the same text gets dumped both for help output and when the command executes; see 7589e8f! Thanks again for catching this.